SeaBattle/BaseShip.cs

```diff
@@ -0,0 +1,148 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+using System.Drawing;
+using System.Reflection;
+using System.Xml.Linq;
+using System.Data.SqlTypes;
+using System.Runtime.CompilerServices;
+
+namespace SeaBattle
+{
+    public class BaseShip
+    { 
+        public List<Deck> Decks { get; set; }
+        private int Durability { get; set; }
+        private int MoveSpeed { get; set; }
+        private int MoveDirectionX { get; set; }
+        private int MoveDirectionY { get; set; }
+        private string Name { get; set; }
+        private int Speed { get; set; }
+        
+        public BaseShip(string Name, List<Point> points) {

```

Comment:
You don't use the Name argument anywhere, you missed the assignment

Reply:
Every Ship has name, thats why Im added Name to ships.

Reply:
Yep, but you are passing this argument to the constructor and then don't assign it anywhere, so it just vanishes.

Reply:
Got it now. Fixed.